mne/transforms.py

```diff
@@ -576,6 +643,7 @@ def get_ras_to_neuromag_trans(nasion, lpa, rpa):
     return trans
 
 
+@deprecated('collect_transforms is deprecated and will be removed in v0.11')
```

Comment:
And this function's only purpose seems to be to make files for `transform_coordinates` (and it's not even listed in our API docs).